DirectEmployers/MyJobs
# Add permissions to analytics report intentions [PD-3017]

## What's this PR do?
Ensures the "Job Analytics" report intention is protected by permissions.
## Screenshots (if appropriate)

## Where should the reviewer start?
`models.py`. The rest is testing or required changes to support the model modifications.
## How should this be manually tested?
Ensure that you can see the "Job Analytics" intention. Remove the "view analytics" permission somehow. Now you should not be able to see it.
## Any background context you want to provide?
